cypress/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig.json",
+  "include": ["../node_modules/cypress", "*/*.ts"],
```

Comment:
surprised by the node_modules inclusion.  Is that for sourcemaps or something?

Reply:
I am not entirely sure, I do see it like this in a lot of other [repos](https://github.com/search?q=%22include%22%3A+%5B%22..%2Fnode_modules%2Fcypress%22%2C+%22*%2F*.ts%22%5D%2C&type=code).

Reply:
I also see it without in several [repos](https://github.com/search?q=%22include%22%3A+%5B%22*%2F*.ts%22%5D&type=code).

Reply:
I'll remove and check that CI still passes.

Reply:
@silasbw stripped down to just what we need in a4ef02554e36f189a38d382dd410fa62b60aed94!